index.html

```diff
@@ -61,6 +61,9 @@
     <h2>Terms and Definitions</h2>
 
     <p><dfn>Deprecated</dfn>. Permitted today, but expected to be prohibited in the future.</p>
+
+    <p><dfn>At-risk</dfn>. Required, only to the extent that implementation experience can be demonstrated; otherwise not
```

Comment:
Doesn't feel quite right to be calling a requirement "at risk" - using a [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method) rating this would be a Should or a Could requirement. In this case we are introducing a classification problem for `#shear` vs `#lineShear` by using the feature designators as requirement indicators. 

I think it would be better if we had a requirement statement like "be able to specify a shear transformation on line areas" instead.